.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": false,
```

Comment:
Can't believe I did this.

Reply:
I truly appreciate you putting this rule in

Reply:
I lost a little bit of my soul.